src/ripple/protocol/impl/BuildInfo.cpp

```diff
@@ -37,6 +37,9 @@ char const* const versionString = "1.8.1"
 // clang-format on
 
 #if defined(DEBUG) || defined(SANITIZER)
+#ifdef GIT_COMMIT_HASH
+    "-" GIT_COMMIT_HASH
+#endif
```

Comment:
Is it really your intention to have this block only added when compiling builds with `DEBUG` or `SANITIZER` support? If not, you should probably move this block up to line 38.